auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -192,13 +193,13 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
     return item.corpType.code === CorpType.NEW_BUSINESS && item.name === item.businessIdentifier
   }
 
-  /** Returns true if the affiliation is approved to start an IA */
-  private isApprovedForIA (business: Business): boolean {
+  /** Returns true if the affiliation is approved to start an IA or Registration */
+  private showUseNameRequest (business: Business): boolean {
```

Comment:
Very small nit, but it's clearer to use "is" or "has" for boolean function names, and "do" or "show" (etc) for action function names.